### DISCUSSION TITLE
No Api keys found in request

### DISCUSSION THREAD

**Discussion by coach7290** [supabase/supabase#3301] [Questions]
Hi Guys, I've started to learn how to integrate supabase with React, using the user management boilerplate. The example worked well, but when I'm trying to create another table to import some datas, I got this error in the console log. What this could be be the issue?

**Comment by dshukertjr** [comment]
Hi @coach7290!

Could you share some code that you have wrote so that we can reproduce the issue? It could be an environment variable issue or something.
